korman/properties/prop_world.py

```diff
@@ -30,9 +30,9 @@ class PlasmaFni(bpy.types.PropertyGroup):
                                     subtype="COLOR")
     fog_method = EnumProperty(name="Fog Type",
                               items=[
-                                     ("linear", "Linear", "Linear Fog"),
-                                     ("exp", "Exponential", "Exponential Fog"),
-                                     ("exp2", "Exponential 2", "Exponential Fog 2"),
+                                     ("linear", "Linear", "Fog Based on Line of Sight"),
+                                     ("exp", "Exponential", "Fog Based on Avatar Position"),
+                                     ("exp2", "Exponential 2", "Exponential Fog With Distance Squared"),
```

Comment:
Better descriptions might be:

* **Linear:** Fog based on linear distance
* **Exponential:** Fog based on exponential distance
* **Exponential Squared:** Fog based on exponential distance squared

All fog is based on the distance from the camera to a vertex, with the following equations:

* **Linear**: `(fogEnd - camToVertexDistance) / (fogEnd - fogStart)`
* **Exponential:** `e ^ -(fogDensity * camToVertexDistance)`
* **Exponential Squared:** `e ^ -((fogDensity * camToVertexDistance) ^ 2)`

Reply:
@dpogue Good enough for me. I'll make the changes.